tests/flow/generics/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -72,7 +72,9 @@ class E<X> extends C<X> {
   //x:X;
   set(x: X): X {
     /*return x;*/ this.x = x;
-    return /*this.x; */ this.get();
+    return (
+      /*this.x; */ this.get()
```

Comment:
This is less precise as it also triggers on this use case. We could fix it by reproducing the logic inside of comments to know whether it's going to be on its own line but I think it's a good trade-off to just have a parenthesis here and make the code simpler.